vgc/autopilot_test.py

```diff
@@ -0,0 +1,36 @@
+"""
+Test for autopilot.py
+"""
+from autopilot import get_roll
+from autopilot import get_pitch
+from autopilot import get_yaw
+from autopilot import get_lat
+from autopilot import get_lon
+from autopilot import get_alt
+from autopilot import connecting_with_autopilot
```

Comment:
**Don't import like this.** Use relative imports.
Import the module and then access each function. 
```python
from . import autopilot

#call cuntion 
autopilot.get_yaw()
```

Reply:
It is fixed :)